package.json

```diff
@@ -10,6 +10,9 @@
     "url": "https://github.com/JedWatson/classnames.git"
   },
   "types": "./index.d.ts",
+  "engines": {
+    "node": "^10 || ^12 || ^14 || ^15"
+  },
```

Comment:
I don't know if we want to limit the `engines` in this way

Reply:
It doesn't prevent the package from being installed to my knowledge. NPM will simply produce a warning.